_posts/2016/2016-05-31-gradle-jenkins-plugin.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: post
+title: "How to empower your Jenkins with Gradle"
+date: "2016-05-31 10:59:11"
```

Comment:
Your post is planned for June 14th, 2016. Nevertheless, if changes occur to the editorial calendar, the 7th of June would be also possible.